src/frameserver/PointGreyCam.cpp

```diff
@@ -491,44 +491,46 @@ void PointGreyCam<T>::setupPixelBinning(size_t x_bin, size_t y_bin)
 //}
 
 // TODO: Required??
-//template <typename T>
-//void PointGreyCam<T>::turnCameraOn()
-//{
-//    // Power on the camera_
-//    const unsigned int k_cameraPower = 0x610;
-//    const unsigned int k_powerVal = 0x80000000;
-//    pg::Error error = camera_.WriteRegister(k_cameraPower, k_powerVal);
-//    if (error != pg::PGRERROR_OK && error != pg::PGRERROR_NOT_IMPLEMENTED)
-//        throw (rte(error.GetDescription()));
-//
-//    unsigned int regVal = 0;
-//    unsigned int retries = 10;
-//
-//    // Wait for camera_ to complete power-up
-//    do {
-//
-//        std::this_thread::sleep_for(std::chrono::milliseconds(100));
-//        error = camera_.ReadRegister(k_cameraPower, &regVal);
-//        if (error == pg::PGRERROR_TIMEOUT) {
-//            // ignore timeout errors, camera_ may not be responding to
-//            // register reads during power-up
-//        } else if (error != pg::PGRERROR_OK) {
-//            throw (rte(error.GetDescription()));
-//        }
-//
-//        retries--;
-//    } while ((regVal & k_powerVal) == 0 && retries > 0);
-//
-//    // Check for timeout errors after retrying
-//    if (error == pg::PGRERROR_TIMEOUT)
-//        throw (rte(error.GetDescription()));
-//}
+template <typename T>
+void PointGreyCam<T>::turnCameraOn()
+{
+    // Power on the camera_
+    const unsigned int k_cameraPower = 0x610;
+    const unsigned int k_powerVal = 0x80000000;
+    pg::Error error = camera_.WriteRegister(k_cameraPower, k_powerVal);
+    if (error != pg::PGRERROR_OK && error != pg::PGRERROR_NOT_IMPLEMENTED)
+        throw (rte(error.GetDescription()));
+
+    unsigned int regVal = 0;
+    unsigned int retries = 10;
+
+    // Wait for camera_ to complete power-up
+    do {
+
+        std::this_thread::sleep_for(std::chrono::milliseconds(100));
+        error = camera_.ReadRegister(k_cameraPower, &regVal);
+        if (error == pg::PGRERROR_TIMEOUT) {
+            // ignore timeout errors, camera_ may not be responding to
+            // register reads during power-up
+        } else if (error != pg::PGRERROR_OK) {
+            throw (rte(error.GetDescription()));
+        }
+
+        retries--;
+    } while ((regVal & k_powerVal) == 0 && retries > 0);
+
+    // Check for timeout errors after retrying
+    if (error == pg::PGRERROR_TIMEOUT)
+        throw (rte(error.GetDescription()));
+}
 
 template <typename T>
 void PointGreyCam<T>::setupAsyncTrigger(int trigger_mode,
                                         bool trigger_rising,
                                         int trigger_pin)
 {
+    std::cout << "Setting up async trigger mode "<< trigger_mode << " \n";
+  
```

Comment:
It's there in the subsequent lines - see line 534 [https://github.com/mmyros/Oat/blob/682330c281c73f6939f64a5d64588d130320c87d/src/frameserver/PointGreyCam.cpp#L534](url)